javascript/javascript3/week3/RewriteTime.js

```diff
@@ -0,0 +1,25 @@
+//1.
+
+function promiseToTimeOut(milisecs) {
+ return new Promise ((resolve) => {

```

Comment:
As the constructor of `Promise` expects you to pass a funciton as a first argument, you can write the whole thing as:

```
const promiseToTimeOut = (milisecs) =>
        new Promise((resolve) => {
          setTimeout(resolve, milisecs); // here we pass resolve directly to the constructor
        });
```